components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/config/ConfigurationFacade.java

```diff
@@ -356,7 +356,9 @@ private String buildAbsoluteUrl(String defaultContext, Supplier<String> getValue
             }
```

Comment:
This method is called for building authentication and recovery endpoint URLs.
<img width="1148" alt="Screenshot 2023-11-24 at 10 05 03" src="https://github.com/wso2/carbon-identity-framework/assets/35717390/c6dfa246-04c6-475a-af6f-5fbbd9138768">